build.gradle.kts

```diff
@@ -142,6 +154,16 @@ tasks {
         pluginClasspath.from.clear()
         pluginClasspath.from(shadowJar)
     }
+    register<Test>("e2eTest") {
+        testClassesDirs = sourceSets.get("e2eTest").output.classesDirs
+        classpath = sourceSets.get("e2eTest").runtimeClasspath
```

Comment:
`sourceSets.e2eTest.runtimeClasspath` known from Groovy would be more readable here.